File: ejercicios/python/ejercicio073.py
def contador_recursivo(numero):
   if (numero == 0):
      return 0
   else:
      return 1 + contador_recursivo(int(numero/10))

if __name__ == "__main__":
   numero = int(input("ingrese el numero : "))
   if (numero >= 0):
      print(str(numero)+" tiene "+str(contador_recursivo(numero))+" digitos")
   else:
      print("Error el numero debe ser positivo")

File: ejercicios/c++/ejercicio073.cpp
#include <iostream>
using namespace std;

int contador_recursivo(int numero){	if(numero == 0)		return 0;	else		return 1 + contador_recursivo(numero/10);}

int main() {	int numero;	cout << "ingrese un numero entero : ";	cin >> numero;	if (numero >= 0)		cout << "el numero " << numero << " tiene " << contador_recursivo(numero) << " digitos " << endl;	else		cout << "El numero debe ser positivo." << endl;	return 0;}
